packages/studio-ui/src/web/utils.test.ts

```diff
@@ -0,0 +1,17 @@
+import { recursiveSearch } from './util'
+
+test('recursiveSearch', () => {
+  const obj = {
+    sub: {
+      id: 1,
+      type: 'image',
+      micro: {
+        name: 'test',
+        type: 'node'
+      }
+    },
+    id: 2,
+    type: 'standard'
+  }
+  expect(recursiveSearch(obj, 'type')).toEqual(['image', 'node', 'standard'])
+})
```

Comment:
Can you add the scenarios where `recursiveSearch` is called with an empty object, undefined, null, a number, ... Also, what if you are searching for a key that does not exist? Make sure to check all cases that you thought of handling when making the tests.